bagitobjecttransfer/recordtransfer/templates/admin/base_site.html

```diff
@@ -29,18 +30,31 @@
             .section:visited {
                 color: var(--secondary) !important;
             }
+
+            #header a:link, #header a:visited, #logout-form button {
+                color: var(--link-fg);
+            }
+
+            .theme-icon-when-light {
+                fill: var(--link-fg); 
+            }
+
+            #user-tools a, #logout-form button {
+                border-bottom: 1px solid rgb(194, 194, 194);
```

Comment:
All the other colours are in hex, this one looks out of place here

```suggestion
                border-bottom: 1px solid #c2c2c2;
```